store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9802,4 +9802,37 @@ TODO: delete them permanently from here
     <defaultCOSValue>TRUE</defaultCOSValue>
     <desc>Whether Powerpaste preference is enabled by user or admin</desc>
 </attr>
+
+<attr id="3086" name="zimbraFeatureChangeUseImapEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited,domainAdminModifiable" since="8.8.15">
+  <defaultCOSValue>FALSE</defaultCOSValue>
+  <desc>
+    whether to allow end user to disable IMAP feature.
+    If it is set to TRUE, IMAP feature is available only when both zimbraImapEnabled and zimbraPrefUseImap are TRUE.
+    If it is set to FALSE, zimbraPrefUseImap is ignored.
+  </desc>
+</attr>
+
+<attr id="3087" name="zimbraFeatureChangeUsePop3Enabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited,domainAdminModifiable" since="8.8.15">
```

Comment:
please make attribute name more intuitive zimbraFeatureChangeUseImapEnabled
**zimbraFeaturePop3Enabled** would be a better choice.

Reply:
if it user preferences  then **zimbraPrefPop3Enabled**

Reply:
Fixed.